src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final Properties DB_PROPERTIES;
+    private static final String CONNECTION_URL = "jdbc:mysql://localhost:3306/demoapp";
```

Comment:
The CONNECTION_URL is incomplete, and it should include the full JDBC URL to the database. For example, 'jdbc:mysql://localhost:3306/databaseName' where 'databaseName' is the name of your database.